mirascope/base/extractors.py

```diff
@@ -125,22 +144,35 @@ def _extract(
         kwargs, return_tool = self._setup(tool_type, kwargs)
 
         class TempCall(call_type):  # type: ignore
-            prompt_template = self.prompt_template
+            prompt_template = """
+            {original_prompt_template}
+
+            {error_message}
```

Comment:
this will include unnecessary additional `\n` characters even if `error_message` is empty